### PR TITLE
[DOC] Review of Event.yml

### PR DESCRIPTION
**Pull request by ypbnv**
Add a missing platform.
Try to explain the exposure of RecurrenceRule.

### CODE REVIEW

File: apidoc/Titanium/Calendar/Event.yml [hunk 1]
```diff
@@ -76,6 +77,8 @@ methods:
         When an event is saved, it is updated in the Calendar database. Any fields you did 
         not modify are updated to reflect the most recent value in the database. If the 
         event has been deleted from the database, it is re-created as a new event.
+
+        On Android, this method only updates the recurrence rule (if set).
```
Comment (hansemannn) [inline apidoc/Titanium/Calendar/Event.yml]: @ypbnv Do we have a parity ticket for this?

File: apidoc/Titanium/Calendar/Event.yml [hunk 2]
```diff
@@ -3,8 +3,8 @@ name: Titanium.Calendar.Event
 summary: An object that represents a single event in a calendar.
 description: |
     The API supports retrieving information about existing events and creating new events.  
-    Be aware that modifying or deleting existing events and creating recurring rules for events 
-    are only supported on iOS.
+    Be aware that modifying, with the exception of adding a new recurrence rule, or deleting
```
Comment (sgtcoolguy) [inline apidoc/Titanium/Calendar/Event.yml]: This sentence feels clunky and is difficult to easily understand at first glance. Perhaps @jawa9000 can help massage the updated text here?

File: apidoc/Titanium/Calendar/Event.yml (hunk 1 above)
Comment (ypbnv) [reply apidoc/Titanium/Calendar/Event.yml]: @hansemannn No, I think. It was added with all other changes for this ticket: https://jira.appcelerator.org/browse/TIMOB-24248

File: apidoc/Titanium/Calendar/Event.yml (hunk 2 above)
Comment (ypbnv) [reply apidoc/Titanium/Calendar/Event.yml]: I was trying to be as minimal as possible. But now that I think of it, a more detailed description may be needed here.

**Pull request merged**